packages/strapi-admin/admin/src/containers/InstalledPluginsPage/Row.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { IconLinks } from '@buffetjs/core';
+import { CustomRow } from '@buffetjs/styles';
+import { useGlobalContext, PopUpWarning } from 'strapi-helper-plugin';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useHistory } from 'react-router-dom';
+import { faTrash, faCog } from '@fortawesome/free-solid-svg-icons';
+import Text from '../../components/Text';
+import LogoContainer from './Logo';
+
+// TODO:
+const PLUGINS_WITH_CONFIG = ['email', 'upload'];
+
+const Row = ({ logo, name, description, isRequired, id, icon, onConfirm }) => {
+  const { currentEnvironment, formatMessage } = useGlobalContext();
+  const [isOpen, setIsOpen] = useState(false);
+  const { push } = useHistory();
+  const links = [];
+
+  if (PLUGINS_WITH_CONFIG.includes(id)) {
+    links.push({
+      icon: <FontAwesomeIcon icon={faCog} />,
+      onClick: () => {
+        const settingsPath = `/plugins/${id}/configurations/${currentEnvironment}`;
+
+        push(settingsPath);
+      },
+    });
+  }
+
+  const handleClickConfirm = () => {
+    handleToggle();
+    onConfirm(id);
+  };
+
+  const handleToggle = () => {
+    setIsOpen(prev => !prev);
+  };
+
+  if (!isRequired) {
+    links.push({
+      icon: <FontAwesomeIcon icon={faTrash} />,
+      onClick: handleToggle,
+    });
+  }
+
+  return (
+    <CustomRow>
+      <td style={{ paddingLeft: 30, paddingTop: 0, width: 100 }}>
+        <LogoContainer>
+          {logo && <img src={logo} alt="icon" />}
+          {!logo && (
+            <div className="icon-wrapper">
+              <FontAwesomeIcon icon={icon} />
+            </div>
+          )}
+        </LogoContainer>
+      </td>
+      <td>
+        <p>
```

Comment:
Why do you need a <p /> here ?

Reply:
I have just taken the exemple implementation of the list component from Buffet.js